pyproject.toml

```diff
@@ -75,6 +75,10 @@ recommended = [
     'pillow',
     'plotly',
 ]
+fastapi = [
+    'bokeh-fastapi == 0.1.0',
```

Comment:
This will only resolve after the initial release, but that should happen before the next panel release anyway. I opted for a hard-pin here since there are no BC guarantees yet for `bokeh-fastapi`.